frontend/src/queries/nodes/DataVisualization/Components/Variables/Variables.tsx

```diff
@@ -305,17 +307,20 @@ export const VariableComponent = ({
     variableOverridesAreSet,
     onRemove,
     variableSettingsOnClick,
+    insightsUsingVariable,
 }: VariableComponentProps): JSX.Element => {
     const [isPopoverOpen, setPopoverOpen] = useState(false)
 
+    let tooltip = `Use this variable in your HogQL by referencing {variables.${variable.code_name}}`
```

Comment:
I think we're [trying to refer to HogQL as simply SQL](https://github.com/PostHog/posthog.com/pull/10652) in most places, so maybe it makes sense to use just 'SQL` here too?

Reply:
Good point. I'll get this changed in a new PR - I'm sure there are other places we're referencing hogql too